### PR TITLE
Update Release Notes

### PR DESCRIPTION
**Pull request by gantunesr**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,32 +1,32 @@
 # Changelog
 
 ## Current Main Branch
+
+## 5.7.0 - Sept 22, 2022
```
Comment (sethkfman) [inline CHANGELOG.md]: This should be 5.8.0

Comment (gantunesr) [reply CHANGELOG.md]: Updated [c85db69](https://github.com/MetaMask/metamask-mobile/pull/5020/commits/c85db69b89690cdd106496c01a930fbec4984a0c)

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,32 +1,32 @@
...
+- [#4430](https://github.com/MetaMask/metamask-mobile/pull/4430): [FIX] Keystone: Pagination and missing addresses
+- [#3438](https://github.com/MetaMask/metamask-mobile/pull/3438): [DEPENDENCIES] Bump metro from 0.59.0 to 0.66.2
+- [#4649](https://github.com/MetaMask/metamask-mobile/pull/4649): [ENHANCEMENT] Url redirection from QR code
+- [#4980](https://github.com/MetaMask/metamask-mobile/pull/4980): [DEPENDENCIES] On-ramp: Add compact payment method selector
 - [#4982](https://github.com/MetaMask/metamask-mobile/pull/4982): [FIX] typo in faucet name
 - [#4983](https://github.com/MetaMask/metamask-mobile/pull/4983): [FIX] Navigation comment spell
 - [#4755](https://github.com/MetaMask/metamask-mobile/pull/4755): [UPDATE] - New template fields added to reflect newer requirements for PRs
-- [#4978](https://github.com/MetaMask/metamask-mobile/pull/4978): Store distinct id for consistency
+- [#4978](https://github.com/MetaMask/metamask-mobile/pull/4978): [ENHANCEMENT] Store distinct id for consistency
 - [#4958](https://github.com/MetaMask/metamask-mobile/pull/4958): [FIX] WebView Origin Whitelist
```
Comment (sethkfman) [inline CHANGELOG.md:20]:
```suggestion
- [#4958](https://github.com/MetaMask/metamask-mobile/pull/4958): [FIX] WebView Origin Allowlist
```

**Review by sethkfman — APPROVED** [review]
LGTM

**Pull request merged**